plugins/opds.koplugin/_meta.lua

```diff
@@ -0,0 +1,6 @@
+local _ = require("gettext")
+return {
+    name = "opds",
+    fullname = _("OPDS"),
+    description = _([[OPDS desc]]),
```

Comment:
NB

Reply:
Something like this?

```suggestion
    description = _([[OPDS allows you to download books from online catalogs.]]),
```